Add zoom feature

jensenchelese opened the pull request:
@PeanutJack I just finished the zoom feature. Can you take a look at the changes and see if you have any edits? Thanks!

PeanutJack commented on client/App.jsx:
```diff
@@ -61,21 +64,44 @@ class App extends React.Component {
       )
+    } else {
+      return (
+        <Element id='zoomedImg' style={{overflow: 'hidden', backgroundImage: 'url(' + this.state.mainImg + ')', backgroundSize: '200%', backgroundRepeat: 'no-repeat'}} onClick={this.zoomFeature} onMouseOver={() => {this.moveZoom()}}>
```
Could this styling be moved to a different variable that inherits styling from Element then adds more based on props passed in? If not then maybe add new lines after commas for readability.

PeanutJack replied:
Could this styling be moved to a different variable that inherits styling from Element then adds more based on props passed in? If not then maybe add new lines after commas for readability.

jensenchelese commented:
@PeanutJack I adjusted the eslint like you suggested. I also adjusted the formatting for the Element tag. I tried to put it in its own variable but then I didn't have access to the state url so I just rearranged the formatting to be cleaner.
